.github/workflows/integration.yml

```diff
@@ -34,10 +36,26 @@ on:
   # manual trigger
   workflow_dispatch:
```

Comment:
Do we anticipate other workflows (i.e. in dbt-core) wanting to run this workflow? Should we also add `workflow_call`?

Reply:
Workflow_call just calls the action code, without actually executing in this repository. We might want to standardize on an action in a central repository, but I'm guessing it wouldn't be in dbt-snowflake. Workflow_dispatch works for cross-repo calls. I'm running these now with a script, but once the individual repository changes are in place I'm going to try to set up an action in dbt-adapters to call all of the individual adapter integration tests and see how that goes.

Reply:
Agreed, `workflow_dispatch` is effectively `workflow_call` + the workflow repo's code in this scenario, which is what we want.